static/directives/repo-view/repo-panel-info.html

```diff
@@ -23,6 +23,12 @@
     </div>
 
     <!-- Builds -->
+    <div class="col-sm-7 builds-list" ng-if="!Features.BUILD_SUPPORT">
```

Comment:
You can remove `builds-list` from this element (and from the next one for docker). It'll decrease its height.